tests/e2e/specs/shopper/mini-cart.test.js

```diff
@@ -218,4 +219,41 @@ describe( 'Shopper → Mini Cart', () => {
 			);
 		} );
 	} );
+
+	describe( 'Cart page', () => {
+		beforeAll( async () => {
+			await shopper.emptyCart();
+		} );
+
+		it( 'Can go to cart page from the Mini Cart Footer', async () => {
+			const productTitle = await page.$eval(
+				'.wc-block-grid__product:first-child .wc-block-components-product-name',
+				( el ) => el.textContent
+			);
+
+			await page.click(
+				'.wc-block-grid__product:first-child .add_to_cart_button'
+			);
+
+			await expect( page ).toMatchElement(
+				'.wc-block-mini-cart__products-table',
+				{
+					text: productTitle,
+				}
+			);
+
+			const cartUrl = await page.$eval(
+				'.wc-block-mini-cart__footer-cart',
+				( el ) => el.href
+			);
+
+			expect( cartUrl ).toMatch( SHOP_CART_PAGE );
+
+			await page.goto( cartUrl, { waitUntil: 'networkidle0' } );
```

Comment:
Is there a specific reason we don't simulate the click on the button? I guess that we should test it. Maybe we could add  `e.preventDefault()` that prevents the default behavior in the future.

Reply:
Clicking directly on a link may cause racing conditions leads to flaky issue sometime. I use goto for more realiable.